test/truncate.jl

```diff
@@ -81,7 +81,8 @@ function verify_and_test(d::UnivariateDistribution, dct::Dict, n_tsamples::Int)
         if !(typeof(d) in [Distributions.Truncated{Distributions.NoncentralChisq{Float64},Distributions.Continuous, Float64},
                            Distributions.Truncated{Distributions.NoncentralF{Float64},Distributions.Continuous, Float64},
                            Distributions.Truncated{Distributions.NoncentralT{Float64},Distributions.Continuous, Float64},
-                           Distributions.Truncated{Distributions.StudentizedRange{Float64},Distributions.Continuous, Float64}])
+                           Distributions.Truncated{Distributions.StudentizedRange{Float64},Distributions.Continuous, Float64},
+                           Distributions.Truncated{Distributions.Rician{Float64},Distributions.Continuous, Float64}])
```

Comment:
Do you know what the problem is here?

Reply:
Yes, needs Dual numbers support for this test.

Reply:
Ah makes sense, then this will be fixed if there is a native Julia implementation of `nchisq` in StatsFuns :+1: